docs/source/reference/envvars.rst

```diff
@@ -452,6 +452,15 @@ GPU support
    indicating that verbose messaging is enabled. This should not need to be
    modified under normal circumstances.
 
+.. envvar:: NUMBA_CUDA_PER_THREAD_DEFAULT_STREAM
+
+   When set to 1, the default stream is the per-thread default stream. When set
+   to 0, the default stream is the legacy default stream. This defaults to 0,
+   for the legacy default stream, and may default to 1 in a future release of
```

Comment:
I think the suggestion might be slightly confusing, because saying "0, the legacy default stream" sounds a bit like stream 0 always refers to the legacy default stream, whereas stream 0 is the legacy default stream or the per thread default stream depending on the setting of this environment variable. How about?

```suggestion
   for the legacy default stream. It may default to 1 in a future release of
```

Reply:
Good point, agree with your proposed change.

Reply:
Change applied.